netbox/dcim/api/views.py

```diff
@@ -432,7 +434,14 @@ def render_config(self, request, pk):
         # Compile context data
         context_data = device.get_config_context()
         context_data.update(request.data)
-        context_data.update({'device': device})
+        context_data.update({'object': device})
```

Comment:
I realize `object` is more standard, but IMO we should keep this as `device` now that the name collision is no longer a concern.